.travis.yml

```diff
@@ -14,15 +11,32 @@ env:
     - PYSAL_PYPI=false PYSAL_PLUS=false
 
 matrix:
-  allow_failures:
-      - python: 3.5
-        env: PYSAL_PYPI=false PYSAL_PLUS=false
-      - python: 3.5
-        env: PYSAL_PYPI=false PYSAL_PLUS=true
-      - python: 3.6
-        env: PYSAL_PYPI=false PYSAL_PLUS=false
-      - python: 3.6
-        env: PYSAL_PYPI=false PYSAL_PLUS=true
+  include:
+    - python: 3.6
+      env: PYSAL_PYPI=false PYSAL_PLUS=false
+    - python: 3.6
+      env: PYSAL_PYPI=false PYSAL_PLUS=true
+    - python: 3.7
+      dist: xenial
+      sudo: true
+      env: PYSAL_PYPI=false PYSAL_PLUS=false
+    - python: 3.7
+      dist: xenial
+      sudo: true
+      env: PYSAL_PYPI=false PYSAL_PLUS=true
+  allowed_failures:
+    - python: 3.6
```

Comment:
The parameters under `allow_failures` are the same as those under `include`?

Reply:
To be honest I am not sure what is going on. Any thought?

Reply:
TravisCI only run the 3.6 and 3.7 tests (which pass) when `include` in included. I have not been able to get `allowed_failures` to kick in yet. I feel like I am probably missing something super simple, but not sure what...

Reply:
s/allowed/allow/

Reply:
@sjsrey your eagle eye has saved my sanity. Is it strange that Travis does not fail or throw a warning when I so clearly has used an incorrect argument?

Reply:
correcting my typo (`allowed_failures` to `allow_failures`) corrected the problem and now "Allow Failures" run as they should, with the right environment variables. The issue now is that the none "Allow Failures" tests are not picking up the correct python version and all are only running with the default 3.6.3.